### ISSUE TITLE
download link to mac version offline

### ISSUE THREAD

**Issue by core-code** [shoes/shoes3#467]
https://shoes.mvmanila.com/public/shoes/shoes-3.3.7-osx-10.10.tgz 

=> 

`Not Found`

**Comment by ccoupe** [comment]
Yup. Nothing is going to happen to Shoes in the future and nobody is using it much now.  Time to close it down.  I moved the downloads from the website. I can make them available to someone else if they want to host the downloads and cgi scripts.

**Comment by core-code** [comment]
thanks for the update.

**Comment by Schwad** [comment]
@ccoupe how's it going? Well, I hope!

I'm leading a team that is exploring an implementation of the Shoes DSL [here](https://github.com/scarpe-team/scarpe). Would you be able to make the downloads available, even just to me so our dev team can use them?

Additionally, I'm willing to host the downloads and CGI scripts as well.

**Comment by ccoupe** [comment]
Hi, @Schwad .  I'm happy to help. It took me a while to find the downloads on the backup server.  The next question is where can I put them? They are too large to email.

**Comment by IanTrudel** [comment]
> Hi, @Schwad .  I'm happy to help. It took me a while to find the downloads on the backup server.  The next question is where can I put them? They are too large to email.

You could use https://wetransfer.com/ for now. Otherwise I can set up a shared folder on my Google Drive.

**Comment by ccoupe** [comment]
I've got 4GB left in my icloud account. Will that work @Schwad ?

2GB in wetransfer.com may not be enough, @IanTrudel . I'm getting used to push 3G files around so I forget that Shoes was small compared to a LLM file. I'm  have free google drive if I set it up.

**Comment by Schwad** [comment]
I've got a google drive folder with more than enough room for it. :)

My email is ***@***.***, if you let me know the best spot
to contact you I'll email you the link to the folder which will give you
rights to upload. Does that work?

**Comment by ccoupe** [comment]
Turns out google drive is easier to use than icloud. 
@Schwad, That email address isn't shown on my browser - but I used the one on your github profile and sentt the links to the 5 Shoes 3.7 distributions. I also have the 3.6 distributions if you want them.  Let me know if that works.

**Comment by Schwad** [comment]
Thanks much, apologies I didn't know Github clears email addresses when I
send this from my email.

I would take the 3.6 distributions as well if that's not too much trouble.
I've got them downloaded and will now set to uploading them to my personal
drive and also using them. Thank you very much for your help here

**Comment by ccoupe** [comment]
> Thanks much, apologies I didn't know Github clears email addresses when I
send this from my email.
I hadn't seen github do that either but I haven't used it much recently.
The 3.6 files are there now and you should have received notification emails.

**Comment by graza** [comment]
I have some really old Shoes Policeman version that I still return to from time to time when I want to hack a small/light UI together.  Unfortunately the gem installation is failing - I think because of the ancient SSL version.

Is there any way you can share the Walkabout Win32 binaries with me?  That might get me going again...

**Comment by graza** [comment]
Thanks a million @ccoupe

**Comment by ccoupe** [comment]
@graza , I don't think I could have tested on Win 11 - I stopped maintaining before then. There is a second theme, more Windows like which might work better if you can figure out how to enable it. Or was it more X11 oriented? I don't remember. Windows was always the more difficult platform - one of several reasons I stopped.
